src/plot_utils.py

```diff
@@ -633,7 +657,7 @@ def zmatrix(data_df, clustermap_kws=None, row_ordering=None,
         df = df.ix[:, col_ordering]
         df = df.ix[row_ordering, :]
         del clustermap_kws['pivot_kws']
-        fig, ax = plt.subplots()
+        _fig, ax = plt.subplots()
```

Comment:
@riastradh-probcomp I'm not sure I understand what you're looking for. This creates an axis. It's a subplot, so if there's an existing figure for those axes to go to, then it returns that, otherwise someone has to create one anyway. Am I misunderstanding?